client/src/components/editor/MediaBlock.js

```diff
@@ -1,111 +0,0 @@
-// @flow
-import React, { Component } from "react"
-import PropTypes from "prop-types"
-import Tooltip from "components/Tooltip"
-import Portal from "components/Portal"
```

Comment:
As MediaBlock and TooltipEntity have been removed, I think components/Portal.js can also be removed, it is not being used somewhere else.

Reply:
Correct, and thanks to your remark I just noticed it was  not the only obsolete file.